main.js

```diff
@@ -1 +1 @@
-console.log('Mighty CIT');
\ No newline at end of file
+var db  = level("./student-db", { valueEncoding: 'json'});
```

Comment:
Reader that don't have know how on level db would not understand this line
you could create a function named connectToDatabase 

```js
function connectToDatabase(connection) {
      return level(connection, { valueEncoding: "json" } );
}

var db = connectToDatabase("./student-db");
```